src/HealthcareApis/HealthcareApis/HealthcareApis/NewAzureRmHealthcareApisService.cs

```diff
@@ -0,0 +1,310 @@
+                    accessPolicies.Add(new ServiceAccessPolicyEntry(objectId));
+                }
+
+                ServicesDescription servicesDescription = new ServicesDescription()
+                {
+                    Kind = GetKind(),
+                    Location = Location,
+                    Tags = this.GetTags(),
+
+                    Properties = new ServicesProperties()
+                    {
+                        AuthenticationConfiguration = new ServiceAuthenticationConfigurationInfo() { Authority = GetAuthority(), Audience = GetAudience(), SmartProxyEnabled = EnableSmartProxy.ToBool() },
+                        CosmosDbConfiguration = new ServiceCosmosDbConfigurationInfo() { OfferThroughput = GetCosmosDBThroughput()},
+                        CorsConfiguration = new ServiceCorsConfigurationInfo() { Origins = CorsOrigin, Headers = CorsHeader, Methods = CorsMethod, MaxAge = CorsMaxAge, AllowCredentials = AllowCorsCredential },
+                        AccessPolicies = accessPolicies
+                    }
+                };
+
+                if (ShouldProcess(this.Name, Resources.createService))
+                {
+
+                    this.EnsureNameAvailabilityOrThrow();
+
+                    var createAccountResponse = this.HealthcareApisClient.Services.CreateOrUpdate(
+                                    this.ResourceGroupName,
+                                    this.Name,
+                                    servicesDescription);
+
+                    var healthCareFhirService = this.HealthcareApisClient.Services.Get(this.ResourceGroupName, this.Name);
+
+                    WriteObject(healthCareFhirService);
+                }
+            });
+        }
+
+        private Kind GetKind()
+        {
+            if(this.Kind == null && this.FhirVersion!=null)
+            {
+                if (FhirVersion.ToLowerInvariant() == "r4")
+                {
+                    return Management.HealthcareApis.Models.Kind.FhirR4;
+                }
+                else if (FhirVersion.ToLowerInvariant() == "stu3")
+                {
+                    return Management.HealthcareApis.Models.Kind.FhirStu3;
+                }
+                else
+                {
+                    throw new PSArgumentException(Resources.createService_InvalidFhirVersionMessage);
+                }
+            }
+            else if(this.Kind== null && this.FhirVersion == null)
+            {
+                return Management.HealthcareApis.Models.Kind.FhirR4;
+            }
+            else if(this.FhirVersion != null)
+            {
+                return parseKindFromVersion(this.FhirVersion);
+            }
+
+            return parseKind(this.Kind);
+        }
+
+        private Kind parseKindFromVersion(string fhirVersion)
+        {
+            return parseKind(FhirVersion);
+        }
+
+        private Kind parseKind(string kind)
+        {
+            if (kind.ToLowerInvariant().Equals("fhir"))
```

Comment:
@poadhika nit: instead of making the call to `ToLowerInvariant()`, you can do the following:

```cs
if (kind.Equals("fhir", StringComparison.OrdinalIgnoreCase))
{
    ...
}
```